lib/qwerty/style.rb

```diff
@@ -0,0 +1,29 @@
+module Qwerty
+  class Style < Ruote::Participant
+    def on_workitem
+      workitem.fields[:style] = {
+         :font => 'Arial',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.